exampleCourse/questions/demo/annotated/engCircuit/server.py

```diff
@@ -1,40 +1,77 @@
 import random
+import io
 
+import matplotlib.pyplot as plt
+from schemdraw import Drawing
+import schemdraw.elements as elm
+
+def file(data):
+    if data["filename"] == "figure.png":
+        drawing = Drawing()
+        R1 = str(data["params"]["R1"]) + " $\\Omega$"
+        R2 = str(data["params"]["R2"]) + " $\\Omega$"
+        R3 = str(data["params"]["R3"]) + " $\\Omega$"
+        Vt = str(data["params"]["Vt"]) + "V"
+        if data["params"]["ask"] == "equivalent resistance $R_T$":
```

Comment:
Instead of checking whether this value is equal to this one specific string, you should store the `whichfig` variable and case on that value.